Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM nginx:stable-alpine
+
+RUN mkdir -p /var/cache/nginx/client_temp /var/run/nginx
+
+RUN chown -R nginx:nginx /var/cache/nginx /var/run/nginx /usr/share/nginx/html
+
+COPY ./nginx.conf /etc/nginx/nginx.conf
+
+COPY ./angular/dist/angular-starter/ /usr/share/nginx/html
+
+USER nginx
```

Comment:
Does your container start nginx by default?

Reply:
Yes. Even though I didn't explicitly used an instruction to start Nginx (CMD/ENTRYPOINT), Nginx still starts by default due to the settings in the base Nginx Docker image

Reply:
Polina your dockerfile should be explicit. What if the base image changes the command tomorrow and it fails? I.e change the location of the configuration file etc.

Explicit is always better than implicit. Good python rule, that we also follow in the team when we work on the solutions.

Reply:
That helps us avoid the "assumptions".

Reply:
Thank you. Fixed